### ISSUE TITLE
ODBC connections

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ox-it/ords#162]
```
ensure databases can be edited by ODBC connections


```

Original issue reported on code.google.com by `Charlie....@gmail.com` on 1 Aug 2013 at 8:09

**Comment by jajwilson** [comment]
I still can't actually establish an ODBC connection from Access even though the new code has been added. I create a project. I add a db. I set the 'enable ODBC access to project' flag to TRUE. I follow the usual Access ODBC connection process, enter the db name, host, username, password, port, and set the SSL mode to 'allow', but when I attempt to connect, I get the following error:

![odbccantconnect](https://cloud.githubusercontent.com/assets/11886974/10312769/808569bc-6c44-11e5-96c9-1abd211ab548.png)
